### DISCUSSION TITLE
Welcome to pixivpy Discussions!

### DISCUSSION THREAD

**Discussion by upbit** [upbit/pixivpy#152] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by takuyatahata** [comment]
I am japanese github user.  just participate in discussion. can you read japanese article？do i need to translate in english and chinese？
translation and reading take a long time.

**Reply by upbit** [reply]
I have no preference, It's up to you.
In addition, I also use Google Translate for Japanese, if you can translate it into English, the communication content may be more accurate

**Reply by takuyatahata** [reply]
thanks.

**Reply by takuyatahata** [reply]
Hey,may i ask you some questions？
this error message occur  with windows command prompt, i do not go further.↓　 mixed japanese sorry.

(venv) c:\>$ git clone https://github.com/addtobasic/auto-download-pixiv-bookmark-images.git                                                                                                                                                 '$' は、内部コマンドまたは外部コマンド、                                                                                                                                                                                                     操作可能なプログラムまたはバッチ ファイルとして認識されていません。

in case without $marks

(venv) c:\>git clone https://github.com/addtobasic/auto-download-pixiv-bookmark-images.git
'git' は、内部コマンドまたは外部コマンド、
操作可能なプログラムまたはバッチ ファイルとして認識されていません。

i would like to  try download with windows command prompt


when git succesfully message？↓

$ git clone https://github.com/addtobasic/auto-download-pixiv-bookmark-images.git
Cloning into 'auto-download-pixiv-bookmark-images'...
remote: Enumerating objects: 45, done.
remote: Counting objects: 100% (45/45), done.
remote: Compressing objects: 100% (31/31), done.
remote: Total 45 (delta 16), reused 37 (delta 9), pack-reused 0
Receiving objects: 100% (45/45), 75.30 KiB | 3.27 MiB/s, done.
Resolving deltas: 100% (16/16), done.

but  the next command↓

$ cd auto-download-pixiv-bookmark-images

$ python3 -m venv venv
Python

$ source venv/bin/activate
bash: venv/bin/activate: No such file or directory

$ (venv) pip install -r requirements.txt
bash: syntax error near unexpected token `pip'

$ cd auto-download-pixiv-bookmark-images
bash: cd: auto-download-pixiv-bookmark-images: No such file or directory

$ python3 download.py
Python

only python message wrong↑　not enough lack of preparement

**Reply by Xdynix** [reply]
Problems:
- The `$` sign usually indicates the start of a command input (as it often appears in many Unix shell). So when you're executing the command, you should copy the text after it.
- On Windows, the activate script of `virtualenv` is in `venv/Scripts/activate`. So you should execute `venv/Scripts/activate` to activate the environment.
- Similarly, on many Unis shell, it will append the environment's name after activate one. So the command you should execute is `pip install -r requirements.txt`.

As a conclusion, the commands you need to run on Windows, with PowerShell or CMD, are probably:
- `git clone https://github.com/addtobasic/auto-download-pixiv-bookmark-images.git`
- `cd auto-download-pixiv-bookmark-images`
- `.\venv\Scripts\activate`
- `pip install -r requirements.txt`
- `python download.py`

**Reply by Xdynix** [reply]
But the "Python" message you got from `python3` related commands are really confusing. I have no idea.

Maybe you need to use `python` instead of `python3`, as Python installation on Windows usually only has `python.exe`.

And you need to make sure it's in you `PATH` environment variables (you can find steps about how to set it, [example](https://www.java.com/en/download/help/path.html)). Assume your Python in installed in `C:\Program Files\Python310`, and you have `C:\Program Files\Python310\python.exe` exisits, then you should add `C:\Program Files\Python310` to the `PATH`. You need to find the true path used when you install Python.

**Reply by takuyatahata** [reply]
thank you for your kindness
